package/gluon-ebtables/files/lib/gluon/ebtables/101-dir-rules

```diff
@@ -1,2 +1,7 @@
 rule 'IN_ONLY --logical-in br-client -i ! bat0 -j DROP'
 rule 'OUT_ONLY --logical-out br-client -o ! bat0 -j DROP'
+
+rule 'OUTPUT -d Multicast --logical-out br-client -o bat0 -j MULTICAST_OUT'
+rule 'FORWARD -d Multicast --logical-out br-client -o bat0 -j MULTICAST_OUT'
+
+rule 'MULTICAST_OUT -p IPv6 --ip6-protocol ipv6-icmp -j ICMPV6_OUT'
```

Comment:
It is not really clear from the name that *ICMPV6_OUT* only applies to multicast packets. Maybe *MULTICAST_OUT_ICMPV6*?